gamification/process/update.py

```diff
@@ -135,6 +136,29 @@ def get_participant_objects():
     return participants
 
 
+def update_participants_data_with_meta_review(participants):
```

Comment:
One idea is that this method would take list of all the participants objects from the
meta-review partcipant model...and then iterate through each of the participant object from the list while filtering out only those participant whose usernames are in [active_newcomer_list](https://github.com/coala/community/blob/master/data/newcomers.py#L9)(as gamifiction system only work for those participant) and then if that participant have `completed meta-review` then get the object of that participant from the gamification participant model and add the meta-review activity for that participant.

Reply:
I think maybe you're getting confused b/w `meta-review Participants` and `gamification Participants`, meta-review system works for all members at coala and gamification system works for only [active newcomers](https://github.com/coala/community/blob/master/data/newcomers.py#L9) at coala.

participants from `get_participant_objects()` are coming from gamification models, so they do not have fields like `pos_in` and `neg_out` etc..so you can't do[1](https://github.com/coala/community/pull/190/files#diff-49eb4e29b7797643ccd41983aeb4c575R148) as @li-boxuan mentioned there.

and where is `meta_review_completed` field? if you want to do something like `participant.meta_review_completed` with participant object you need a field of that name to the `Participant` model. I think you don't need to do something like that, when you find that a participant has completed meta-review, just add the `meta-review completed` activity for that participant by using [`add_activity`](https://github.com/coala/community/blob/master/gamification/models.py#L135) method.

Maybe rebase this commit on top of https://github.com/coala/community/pull/152 so that this pr passes CI and we can see you changes running.
> I'm testing these changes on another branch so that while pushing the changes, my PR doesn't gets bad with Travis test failing messages